cmd/ethrex_l2/src/commands/stack.rs

```diff
@@ -164,6 +191,83 @@ impl Command {
                     println!("Aborted.");
                 }
             }
+            Command::BlobsSaver {
+                l1_eth_rpc,
+                l1_beacon_rpc,
+                contract_address,
+                data_dir,
+            } => {
+                create_dir_all(data_dir.clone())?;
+
+                let eth_client = EthClient::new(l1_eth_rpc.as_str());
+                let beacon_client = BeaconClient::new(l1_beacon_rpc);
+
+                // Keep delay for finality
+                let mut current_block = eth_client.get_block_number().await? - U256::from(64);
```

Comment:
Could `eth_client.get_block_number()` be less than `64`?

Reply:
I think it's safe to use u64 here, consider the following:
- The latest block on mainnet is currently [22046844](https://etherscan.io/block/22046844)
-  $log2(22046844] = 24.39$
We're still a long way from reaching 2**64 blocks.

Reply:
Done [d761483](https://github.com/lambdaclass/ethrex/pull/2194/commits/d761483d92d5b81b298fc1a9a18526cccc5e2523)